src/test/scala/com/fulcrumgenomics/alignment/NeedlemanWunschAlignerTest.scala

```diff
@@ -278,7 +278,10 @@ class NeedlemanWunschAlignerTest extends UnitSpec {
     result.score shouldBe 4
   }
 
-  it should "fail if the target sequence is shorter than the query sequence" in {
-    an[Exception] shouldBe thrownBy { NeedlemanWunschAligner(1, -1, -3, -1, mode=Glocal).align("ACGTACGTACGT", "ACGT") }
+  it should "align a longer query to the whole target with an insertion" in {
```

Comment:
`align a longer query with an insertion to the whole target`?  It have to convince myself an insertion is always query relative to target otherwise.

Reply:
I'll add some commas ;)